gear/gear/metrics.py

```diff
@@ -1,10 +1,13 @@
 from aiohttp import web
 import prometheus_client as pc  # type: ignore
 from prometheus_async.aio import time as prom_async_time  # type: ignore
+from hailtop.utils import time_msecs
 
 REQUEST_TIME = pc.Summary('http_request_latency_seconds', 'Endpoint latency in seconds', ['endpoint', 'verb'])
 REQUEST_COUNT = pc.Counter('http_request_count', 'Number of HTTP requests', ['endpoint', 'verb', 'status'])
 CONCURRENT_REQUESTS = pc.Gauge('http_concurrent_requests', 'Number of in progress HTTP requests', ['endpoint', 'verb'])
+SQL_QUERY_COUNT = pc.Counter('sql_query_count', 'Number of SQL Queries', ['query_name'])
+SQL_QUERY_LATENCY = pc.Summary('sql_query_latency', 'SQL Query latency in seconds', ['query_name'])
```

Comment:
Let's make the naming consistent with the above metrics, so change `sql_query_latency` to `sql_query_latency_seconds`.